doc/neptune-metadata.rst

```diff
@@ -84,7 +84,15 @@ Type
 Meaning
   Defines which types of auto-pull-requests the repository desires.
   Values can be specified for ``default`` and ``override``.
-  Possible values are ``none``, ``patch``, ``minor``, ``major``
+  Possible values are ``none``, ``patch``, ``minor``, ``major``.
+Default Value
+  If the repository is a library (``library: true``)
+  the default is ``patch``. Otherwise it will be ``minor``.
+
+  The motivation for this is that libraries need to ensure that they remain
+  compatible with older dependency versions: non-`patch` submodule updates may
+  cause breaking changes that force downstream users to update their own
```

Comment:
This is incorrect wording I think -  minor updates do not cause breaking changes on their own (by neptune spec). What instead happens is that minor updates may result in library starting to use new features available only in more recent submodule version and _that_ will be a breaking change for downstream.

Reply:
As @joseph-wakeling-sociomantic suggested this wording, maybe he can also suggest updated version with this in mind? :P

Reply:
@joseph-wakeling-sociomantic  ?

Reply:
How about:

```
The motivation for this is that libraries need to ensure that they remain
compatible with older dependency versions.  If a dependency is updated
to a new minor release this is not breaking in itself, but it makes it
possible to accidentally introduce changes dependent on new features,
without this being caught by CI.  This _would_ be breaking change, that
would impact on any downstream project still using an earlier version of
the dependency.  Restricting library submodule updates to only patch
releases should prevent this.
```

Reply:
seems good to me. Waiting for someone else to confirm

Reply:
:+1: